pkg/rke/k8s_version_info.go

```diff
@@ -666,6 +669,10 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.8.0-patch0",
 			MinRKEVersion:     "1.5.0-rc0",
 		},
+		"v1.28": {
+			MinRancherVersion: "2.8.0-patch0",
```

Comment:
1.28 wouldn't be available out of band, so this will only be available with next rancher release. MinRancherVersion would be `2.8.3-patch0`

Reply:
Updated